spec/services/hearing_day_room_assignment_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+describe HearingDayRoomAssignment do
+  let(:service) do
+    described_class.new(
+      request_type: request_type,
+      assign_room: assign_room,
+      scheduled_for: scheduled_for,
+      room: room
+    )
+  end
+  let(:request_type) { nil }
+  let(:assign_room) { true }
+  let(:scheduled_for) { Time.zone.today }
+  let(:room) { "1" }
+
+  context ".available_room" do
+    subject { service.available_room }
+
+    context "for a central docket" do
+      let(:request_type) { HearingDay::REQUEST_TYPES[:central] }
+
+      context "no rooms are assigned" do
+        it { expect(subject).to be("2") }
+      end
+
+      context "there is already a docket for room 2" do
+        let!(:existing_hearing_day) do
+          create(
+            :hearing_day,
+            scheduled_for: scheduled_for,
+            request_type: request_type,
+            room: "2"
+          )
+        end
+
+        it { expect(subject).to be_nil }
+      end
+    end
+
+    context "for a video docket" do
+      let(:request_type) { HearingDay::REQUEST_TYPES[:central] }
```

Comment:
```suggestion
      let(:request_type) { HearingDay::REQUEST_TYPES[:video] }
```

This should be video, and you'll have to pass a regional office when you create the hearing day.